Task2&3/sample.txt

```diff
@@ -0,0 +1,208 @@
+version 1.0
```

Comment:
Why is this a `.txt` file?

Reply:
Actually it's not relevant to the work I did this week. These are from a long time back. Should have deleted them :/